SAMtags.tex

```diff
@@ -246,13 +247,17 @@ \subsection{Barcodes}
 \end{description}
 
 \subsection{Original data}
-
 \begin{description}
+\item[OA:Z:\tagregex{{\tt (}\emph{rname}{\tt ,}\emph{pos}{\tt ,}\emph{strand}{\tt ,}\emph{CIGAR}{\tt ,}\emph{mapQ}{\tt ,}\emph{NM}{\tt ;)}+}]
+Original alignment of template prior to realignment/unalignment by subsequent tool. 
+Unknown values may be dropped, but commas should remain for parsing. 
+In the presence of an {\tt OA} tag, a subsequent tool should append the current information after the semicolon, not replace its content.
+
 \item[OC:Z:\tagvalue{cigar}]
-Original CIGAR, usually before realignment.
+Deprecated, use {\tt OA instead}. Original CIGAR, usually before realignment.
```

Comment:
Write these after the “Original CIGAR etc” text.  On another line would make looking at the diffs better.  Fix the formatting.